### PR TITLE
Add sentiment-analysis client related code

### PR DESCRIPTION
**Pull request by ayeshLK**
Part of: [#4615](https://github.com/ballerina-platform/ballerina-standard-library/issues/4615)

### CODE REVIEW

File: sentiment_analysis_client/Package.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Sentiment analysis client offers the client functionalities to establish connections with the sentiment API endpoint and retrieve the relevant analysis 
+results for a given content.
+
+### Create sentiment analysis client
+
+An `sentiment.analysis:Client` can be created as follows.
+```ballerina
+import balguides/sentiment.analysis;
+
+public function main() return error? {
```
Comment (dilanSachi) [inline sentiment_analysis_client/Package.md:16]:
```suggestion
public function main() returns error? {
```

File: sentiment_analysis_client/Module.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Sentiment analysis client offers the client functionalities to establish connections with the sentiment API endpoint and retrieve the relevant analysis 
+results for a given content.
+
+### Create sentiment analysis client
+
+An `sentiment.analysis:Client` can be created as follows.
+```ballerina
+import balguides/sentiment.analysis;
+
+public function main() return error? {
```
Comment (dilanSachi) [inline sentiment_analysis_client/Module.md:16]:
```suggestion
public function main() returns error? {
```

File: sentiment_analysis_client/client.bal [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                httpClientConfig.secureSocket = check config.secureSocket.ensureType(http:ClientSecureSocket);
+            }
+            if config.proxy is http:ProxyConfig {
+                httpClientConfig.proxy = check config.proxy.ensureType(http:ProxyConfig);
+            }
+            if config.auth is http:ClientAuthConfig {
+                httpClientConfig.auth = check config.auth.ensureType();
+            }
+        }
+        http:Client httpEp = check new (serviceUrl, httpClientConfig);
```
Comment (dilanSachi) [inline sentiment_analysis_client/client.bal:36]:
```suggestion
        Self.clientEp = check new (serviceUrl, httpClientConfig);
```
Cant we directly do this?

File: sentiment_analysis_client/client.bal [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        self.clientEp = httpEp;
+        return;
+    }
+    #
+    # + return - Created
```
Comment (dilanSachi) [inline sentiment_analysis_client/client.bal:41]: Api doc is not complete

File: sentiment_analysis_client/types.bal [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    # Proxy server password
+    @display {label: "", kind: "password"}
+    string password = "";
+|};
+
+public type Post record {
+    string text;
+};
+
+public type Probability record {
```
Comment (dilanSachi) [inline sentiment_analysis_client/types.bal:65]: Missing docs

File: sentiment_analysis_client/utils.bal [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import ballerina/url;
+
+# Get Encoded URI for a given value.
+#
+# + value - Value to be encoded
+# + return - Encoded string
+isolated function getEncodedUri(anydata value) returns string {
+    string|error encoded = url:encode(value.toString(), "UTF8");
+    if (encoded is string) {
```
Comment (dilanSachi) [inline sentiment_analysis_client/utils.bal:9]:
```suggestion
    if encoded is string {
```

File: sentiment_analysis_client/client.bal (hunk 3 above)
Comment (ayeshLK) [reply sentiment_analysis_client/client.bal:36]: This is a generated code using ballerina open-api tool. So, for time-being lets keep it as it is.

File: sentiment_analysis_client/client.bal (hunk 4 above)
Comment (ayeshLK) [reply sentiment_analysis_client/client.bal:41]: ditto

File: sentiment_analysis_client/types.bal (hunk 5 above)
Comment (ayeshLK) [reply sentiment_analysis_client/types.bal:65]: ditto

File: sentiment_analysis_client/utils.bal (hunk 6 above)
Comment (ayeshLK) [reply sentiment_analysis_client/utils.bal:9]: ditto

**Review by dilanSachi — APPROVED** [review]
LGTM

**Pull request merged**